stackslib/src/net/api/getsortition.rs

```diff
@@ -302,7 +373,7 @@ impl HttpResponse for GetSortitionHandler {
         preamble: &HttpResponsePreamble,
         body: &[u8],
     ) -> Result<HttpResponsePayload, Error> {
-        let sortition_info: SortitionInfo = parse_json(preamble, body)?;
+        let sortition_info: Vec<SortitionInfo> = parse_json(preamble, body)?;
```

Comment:
Can you add test coverage to `stacksilb/src/net/api/tests/getsortition.rs`?

Reply:
Added in 5d860c60b7b2cdb6c69c9e1466028904d240394a